FluidNC/src/Motors/RcServo.cpp

```diff
@@ -92,8 +90,8 @@ namespace MotorDrivers {
             float home_time_sec = (axisConfig->_maxTravel / axisConfig->_maxRate * 60 * 1.1);  // 1.1 fudge factor for accell time.
 
             _disabled = false;
-            set_location();                    // force the PWM to update now
-            vTaskDelay(home_time_sec * 1000);  // give time to move
```

Comment:
this was probably a previous minor bug that should have been `vTaskDelay(home_time_sec * 1000 / portTICK_PERIOD_MS);`